main.tf

```diff
@@ -12,9 +12,9 @@ resource "openstack_compute_keypair_v2" "keypair" {
   public_key = file(var.ssh_public_key)
 }
 
-resource "openstack_compute_instance_v2" "kayobe-seed" {
-  name         = var.seed_vm_name
-  flavor_name  = var.seed_vm_flavor
+resource "openstack_compute_instance_v2" "ansible-control" {
```

Comment:
I think seed was more accurate. We're not currently using this host, but in future we might put some services on it that normally go on the seed.

The ansible control host typically would be where you run terraform from, or a VM not managed by terraform.